src/lib/containers/oauth/OAuthManagement.tsx

```diff
@@ -33,7 +49,17 @@ export const OAuthManagement: React.FunctionComponent = () => {
                   {item.verified ? 'Yes' : 'SUBMIT_VERIFICATION_PLACE_HOLDER'}
                 </td>
                 <td>GENERATE_PLACEHOLDER</td>
-                <td>ACTIONS_PLACEHOLDER</td>
+                <td>
+                  <button
+                    onClick={() => {
+                      setIsShowingCreateClientModal(true)
+                      setSelectedClient(item)
+                      setIsEdit(true)
+                    }}
+                  >
+                    EDIT
+                  </button>
```

Comment:
Just for style consistency with other table components:

```tsx
import { Button } from 'react-bootstrap'
```
```suggestion
                  <Button
                  variant="outline"
                  size="sm"
                    onClick={() => {
                      setIsShowingCreateClientModal(true)
                      setSelectedClient(item)
                      setIsEdit(true)
                    }}
                  >
                    EDIT
                  </Button>
```